src/services/users/users.hooks.js

```diff
@@ -2,15 +2,22 @@ const { authenticate } = require('@feathersjs/authentication').hooks;
 
 const { hashPassword, protect } = require('@feathersjs/authentication-local').hooks;
 
+const checkPermissions = require('feathers-permissions');
+
+const { adminOrOwner, owner, preventChangesIfNotOwner } = require('../utils.hooks');
```

Comment:
Use names in imperative mode. I.e. checkAdminOrOwner instead of adminOrOwner and so on.